code/game/objects/structures/crates_lockers/closets/coffin.dm

```diff
@@ -5,6 +5,25 @@
 	icon_closed = "coffin"
 	icon_opened = "coffin_open"
 
+/obj/structure/closet/coffin/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(src.opened)
+		if(istype(W, /obj/item/weapon/grab))
+			var/obj/item/weapon/grab/G = W
+			src.MouseDrop_T(G.affecting, user)      //act like they were dragged onto the closet
+			return 0
+		if(istype(W,/obj/item/tk_grab))
+			return 0
+		if(!dropsafety(W))
+			return
+		usr.drop_item()

```

Comment:
Usage of `usr` in a non-top level proc. Please use `user`. Refer [here](https://github.com/Aurorastation/Aurora.3/blob/master/.github/CONTRIBUTING.md#regarding-the-variable-usr) as to why and how.